torch/library.py

```diff
@@ -553,6 +554,82 @@ def register(func):
         return register(func)
 
 
+def register_autograd(op: _op_identifier, setup_context_fn: Callable, backward_fn: Callable, /, *, lib=None) -> None:
+    r"""Register a backward formula for this custom op.
+
+    In order for an operator to work with autograd, you need to register
+    a backward formula. There are two pieces to this:
+    1. You must tell us what we need to save from the forward pass for
```

Comment:
Is setup_context actually a must? I would suggest
1. you must tell us how to compute gradients...
2. If you need any of the value from the forward there, you can use setup_ctx to ... 

WDYT?

Reply:
It's not a must, but most people need it. I'll update the wording to that, but I'm not sure if I should switch the order of the args to register_autograd (it's fine if the user passes None as the setup_context)